lib/friends/commands/remove.rb

```diff
@@ -11,6 +11,15 @@
     end
   end
 
+  remove.desc "Removes an alias from a location"
+  remove.arg_name "LOCATION ALIAS"
+  remove.command :alias do |remove_location_alias|
+    remove_location_alias.action do |_, _, args|
+      @introvert.remove_location_alias(name: args.first, nickname: args[1])
```

Comment:
I noticed that this (and the same with adding/removing nicknames) doesn't use `.to_s` but the add does. I assume that's to handle `nil` values, which makes me wonder how this behaves with no arguments passed (or only one). Does it break? Are the `.to_s` calls not needed? Or am I missing something?

Reply:
Oh, that happens when you work to much based on the more similar command 😅. You're correct. The add methods used it and provide better error handling than all of the remove methods. `remove tag` also seems to avoid the additional check.   

It didn't seems to directly break something, but it handles the missing input not as good as the `add alias` method. This could be the reason that it was not noticed before.

For the case that both arguments are missing, somehow the error from `regex.match(nil)` is catch somewhere. At least it doesn't kill the method `thing_with_name_in`. For this case i'm not sure why it works quite normal. 

If the location is given, but not the alias than the `include?` check will fail.  

----



I will add the two checks, like in the `add alias` function
```ruby
raise FriendsError, "Expected \"[Location Name]\" \"[Alias]\"" if name.empty?
raise FriendsError, "Alias cannot be blank" if nickname.empty?
```
in addition to the `.to_s` call to create the same behavior as with the `add alias` method.

Similar that should be done for nickname. Also for the additional tests for that behavior (no argument given, only friends name given).